docs/installation/config.rst

```diff
@@ -84,12 +84,6 @@ Other settings
   sent to the Notificaties API for operations on the Object endpoint.
   Defaults to ``True`` for the ``dev`` environment, otherwise defaults to ``False``.
 
-* ``TWO_FACTOR_FORCE_OTP_ADMIN``: Enforce 2 Factor Authentication in the admin or not.
```

Comment:
Yes, I haven't added it here (same as in Openforms), as it should only be used during development. the dev settings will use a default value of `True`

Reply:
I believe we should keep the possibility to switch off 2fa with env vars, because removing it is a breaking change and unexpected for clients who don't use it
Could you please add an env var for it? 

And could you add a note in the changelog for the future version that the env vars for 2fa are changed and it's planned to make 2fa required for everyone? 
Sjoerd is not happy then we introduce configuration changes without an explicit note.